Dockerfile.windows

```diff
@@ -1,17 +1,24 @@
-FROM microsoft/dotnet:1.1-sdk-msbuild
+# escape=`
+FROM microsoft/dotnet:2.0-sdk-nanoserver
```

Comment:
@natemcmaster I just changed image.
```
$ git diff
diff --git a/Dockerfile.windows b/Dockerfile.windows
index 3f9e4f5..b91b9ff 100644
--- a/Dockerfile.windows
+++ b/Dockerfile.windows
@@ -1,5 +1,5 @@
 # escape=`
-FROM microsoft/dotnet:2.0-sdk-nanoserver
+FROM microsoft/aspnetcore-build-nightly:2

 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
```

Even after change, I got following error. It looks like version miss match.
```
C:\MusicStore\samples\MusicStore\MusicStore.csproj : error NU1605:  MusicStore (>= 1.0.0) -> Microsoft.NETCore.App (>= 2.0.0-preview3-25419-01)
  Generating MSBuild file C:\MusicStore\samples\MusicStore\obj\MusicStore.csproj.nuget.g.props.
  Generating MSBuild file C:\MusicStore\samples\MusicStore\obj\MusicStore.csproj.nuget.g.targets.
  Restore failed in 33.24 sec for C:\MusicStore\samples\MusicStore\MusicStore.csproj.
ERROR: Service 'web' failed to build: The command 'powershell -Command $ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue'; dotnet restore --runtime win10-x64 ./samples/MusicStore' returned a non-zero code: 1
```

full log
https://gist.github.com/takekazuomi/4a04f7075825bb3795f905c7c0301d4a

Can you help me?